test/chunkserver/trash_test.cpp

```diff
@@ -594,14 +593,30 @@ TEST_F(TrashTest, test_chunk_num_statistic) {
     std::vector<std::string> logfiles2{};
 
     // (1) chunk exists when init
+    //  trash/
```

Comment:
Not include the scenario bug triggered? Like '/data/chunkserver1/recycler/8589936511.1625810976/raft_snapshot/temp/data/chunk_10000'

Reply:
done